ldes-server-infra-postgres/postgres-fragmentation-repository/src/main/java/be/vlaanderen/informatievlaanderen/ldes/server/fragmentation/entity/FragmentEntity.java

```diff
@@ -0,0 +1,107 @@
+package be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.entity;
+
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.FragmentPair;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.LdesFragmentIdentifier;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.TreeRelation;
+import be.vlaanderen.informatievlaanderen.ldes.server.domain.model.ViewName;
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.converter.FragmentPairConverter;
+import be.vlaanderen.informatievlaanderen.ldes.server.fragmentation.entities.Fragment;
+import jakarta.persistence.*;
+
+import java.time.LocalDateTime;
+import java.util.ArrayList;
+import java.util.List;
+
+@SuppressWarnings({"java:S1068", "java:S107"})
+@Entity
+@Table(name = "fragmentation_fragment", indexes = {
+		@Index(columnList = "viewName"),
+		@Index(columnList = "collectionName"),
+		@Index(columnList = "deleteTime"),
+		@Index(name = "root_of_view", columnList = "root, viewName"),
+		@Index(name = "immutable_with_parent", columnList = "immutable, parentId")
+})
+public class FragmentEntity {
+	@Id
+	private String id;
+	private Boolean root;
+	private String viewName;
+	@Convert(converter = FragmentPairConverter.class)
+	@Column(name = "fragmentations", columnDefinition = "text")
+	private List<FragmentPair> fragmentPairs;
+	private Boolean immutable;
+	private String parentId;
+	private Integer nrOfMembersAdded;
+	@ElementCollection(fetch = FetchType.EAGER)
+	@CollectionTable(
+			name = "fragmentation_fragment_relations",
+			joinColumns = @JoinColumn(name = "fragment_id")
+	)
+	private List<TreeRelationEntity> relations;
+	private String collectionName;
```

Comment:
I know the field `collectionName` is copy pasted from the Mongo impl, but do we need to save the collectionName separately, as the viewName is already stored as `collectionName/viewName`, as `fragment.getFragmentId().getViewName().asString()` returns it in that format. 
I also don't see any usage or getter for the collectionName

Reply:
since we use this field for queries, we should keep it